Comments Test/someVC.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  someVC.swift
+//  Comments Test
+//
+//  Created by Roman on 2022-09-23.
+//
+
+import UIKit
+func isIPhone() -> Bool {
+    return UIDevice.current.userInterfaceIdiom == .phone
+}
+class someVC: UIViewController,UICollectionViewDelegate,UICollectionViewDelegateFlowLayout,UICollectionViewDataSource {
+    @IBOutlet weak var collectionViewTopConstraint: NSLayoutConstraint!
+    @IBOutlet var detailViewWidthConstraint: NSLayoutConstraint!
+    @IBOutlet var collectionView: UICollectionView!
+    @IBOutlet var detailView: UIView!
+    var detailVC : UIViewController?
+    var dataArray : [Any]?
+    override func viewWillAppear(_ animated: Bool) {
+        fetchData()
+        self.collectionView.dataSource = self
+        self.collectionView.delegate = self
+        self.collectionView.register(UINib(nibName: "someCell",
+                                           bundle: nil), forCellWithReuseIdentifier: "someCell")
+        
+        self.navigationItem.rightBarButtonItem = UIBarButtonItem.init(title:NSLocalizedString("Done&quot", comment: ""), style: .plain, target: self,
+                                                                      action: #selector(dissmissController))
+    }
+    @objc func dissmissController () {}
+    func fetchData() {
+        let url = URL(string: "testreq")!
+        let task = URLSession.shared.dataTask(with: url) { [self](data, response, error) in
+            self.dataArray = data as? [Any]
+            self.collectionView.reloadData()
+        }
+        task.resume()
+    }
+    
+    @IBAction func closeshowDetails () {
```

Comment:
please fix case, and follow naming convention and rename it to more suitable tense like: `closeShowDetails`